ext/dapr-ext-workflow/dapr/ext/workflow/aio/dapr_workflow_client.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Any, Optional, TypeVar
+
+import durabletask.internal.orchestrator_service_pb2 as pb
+from dapr.ext.workflow.logger import Logger, LoggerOptions
+from dapr.ext.workflow.util import getAddress
+from dapr.ext.workflow.workflow_context import Workflow
+from dapr.ext.workflow.workflow_state import WorkflowState
+from durabletask.aio import client as aioclient
+from grpc.aio import AioRpcError
+
+from dapr.clients import DaprInternalError
+from dapr.clients.http.client import DAPR_API_TOKEN_HEADER
+from dapr.conf import settings
+from dapr.conf.helpers import GrpcEndpoint
+
+T = TypeVar('T')
+TInput = TypeVar('TInput')
+TOutput = TypeVar('TOutput')
+
+
+class DaprWorkflowClientAsync:
```

Comment:
I'd call this just `DaprWorkflowClient`, without the async, same as we do with `DaprClient` in both `dapr.aio.clients` and `dapr.clients`